pyproject.toml

```diff
@@ -69,14 +70,16 @@ multilingual = ["nagisa>=0.2.7", "jieba>=0.42.1", "pycountry"]
 openai = ["openai==1.3.9", "tiktoken"]
 optimum = ["optimum[openvino]"]
 promptsource = ["promptsource>=0.2.3"]
-sentencepiece = ["sentencepiece>=0.1.98", "protobuf>=4.22.1"]
+sentencepiece = ["sentencepiece>=0.1.98"]
```

Comment:
is there a reason the `sentencepiece` dependency was changed here?

Reply:
I think there was a version conflict when I was running the GHA - is sentencepiece tested in automation? It would be nice to lower the version requirement if possible

Reply:
Got it! Sounds good to me. Protobuf versions can be often pretty finicky.